### DISCUSSION TITLE
NameError: name 'comfy_ui_hash' is not defined

### DISCUSSION THREAD

**Discussion by EddieSoloProjects** [comfyanonymous/ComfyUI#2262] [Q&A]
All the sudden Comfy Manager doesn't work and getting a NameError at startup. I have no clue hoe to fix this

**Comment by ltdrdata** [comment]
This is an issue that should be reported in the ComfyUI-Manager repository.

It seems that this error occurred in a specific environment. Please update ComfyUI-Manager to the latest version.

Navigate to `ComfyUI/custom_nodes/ComfyUI-Manager` in the terminal and execute `git pull` to update.
